### PR TITLE
fakeroute: init at 0.3

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
See issue #20181.

###### Things done

- [x] Tested using sandboxing
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using (new package)
- [x] Tested execution of all binary files
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Things to do

 - [x] Figure out the license

### CODE REVIEW

File: pkgs/tools/networking/fakeroute/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    sha256 = "1sp342rxgm1gz4mvi5vvz1knz7kn9px9s39ii3jdjp4ks7lr5c8f";
+  };
+
+  meta = with stdenv.lib; {
+    description = ''
+      Makes your machine appear to be anywhere on the internet
+      to any host running a (UDP) unix traceroute
+    '';
+    homepage = https://moxie.org/software/fakeroute/;
+    #license = licenses.???;
```
Comment (fpletz) [inline pkgs/tools/networking/fakeroute/default.nix]: It's `licenses.bsd3`. In the source files you can see a license text on the top which is the [BSD 3-Clause License](https://opensource.org/licenses/BSD-3-Clause). Though the third clause is oddly numbered as fourth. :goat:

**Review by fpletz — CHANGES_REQUESTED** [review]
LGTM apart from the license. :+1:

**Review by fpletz — APPROVED** [review]

**Comment by rnhmjoj** [comment]
It doesn't seem to compile on darwin. Should I set platform.linux?

**Comment by fpletz** [comment]
@rnhmjoj Yes, please. Linux is okay for now.

**Review by fpletz — APPROVED** [review]

**Pull request merged**